docs/sources/flow/reference/cli/run.md

```diff
@@ -75,3 +77,17 @@ other agent nodes; if this is empty it will attempt to find a suitable address
 to advertise from a list of default network interfaces. The agent must be
 reachable over HTTP on this address as communication happens over the agent's
 HTTP server.
+
+## Configuration conversion (beta)
+
+When the `--config.format` command-line argument is provided, Grafana
+Agent will convert the config file from the source format to River and
+immediately start running with it. This leverages the same converter API
+described in the [grafana-agent convert][] docs.
+
+If the `--config.bypass-conversion-warnings` command-line argument is also provided,
+Grafana Agent will ignore any warnings provided by the converter. This should
+be used with caution since the resulting conversion is not equivalent to the
+original config.
```

Comment:
```suggestion
If you also use the `--config.bypass-conversion-warnings` command-line argument,
Grafana Agent will ignore any warnings from the converter. Use this
argument with caution because the resulting conversion is not equivalent to the
original configuration.
```